lib/puppet/ssl/ssl_provider.rb

```diff
@@ -62,8 +62,10 @@ def create_root_context(cacerts:, crls: [], revocation: Puppet[:certificate_revo
   #   `private_key`.
   # @api private
   def create_context(cacerts:, crls:, private_key:, client_cert:, revocation: Puppet[:certificate_revocation])
-    raise ArgumentError, "Private key is required" unless private_key
-    raise ArgumentError, "Client cert is required" unless client_cert
+    raise ArgumentError, "CA certs are missing" unless cacerts
```

Comment:
Realistically, these messages should only ever show up for developers, correct? Just wondering how concerned we should be about having them be translated strings.

Reply:
yeah, but it'd be good to fix that